com.unity.multiplayer.mlapi/Runtime/Messaging/RpcQueue/RpcQueueContainer.cs

```diff
@@ -771,49 +771,7 @@ public RpcQueueContainer(NetworkManager networkManager, uint maxFrameHistory = 0
 
 
 #if UNITY_EDITOR || DEVELOPMENT_BUILD
-        /// <summary>
```

Comment:
Although I like the removal, does it tie in with the rest of the PR in some way?

Reply:
Will discuss in slack DM.  :)